scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+"""
+
+Simple script to populate CESM test database with test results.  This can be 
+run with out any knowledge of CIME.
+
+"""
+
+import os
```

Comment:
Should use 
from Tools.standard_script_setup import *
which will import argparse and glob as well as some standard arguments.